.github/workflows/test.yml

```
@@ -0,0 +1,20 @@
+name: test
+
+on:
+  push:
+    branches: [proposed_change_not_compatible]
+  pull_request:
+    branches: [proposed_change_not_compatible]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
+      - run: npm i
```

```suggestion
      - run: npm install
```
plz dont use shorthand